src/p_map.c

```diff
@@ -1672,7 +1672,7 @@ fixed_t P_AimLineAttack(mobj_t *t1,angle_t angle,fixed_t distance,int mask)
 
   // can't shoot outside view angles
 
-  if (t1->player && direct_vertical_aiming)
```

Comment:
How about a named constant instead of 1? And I guess it should be `mask & constant` instead of an equality, for consistency, if this is indeed a variable holding flags.